src/esp/gfx/RenderTarget.cpp

```diff
@@ -37,7 +38,7 @@ namespace gfx {
 
 const Mn::GL::Framebuffer::ColorAttachment RgbaBuffer =
     Mn::GL::Framebuffer::ColorAttachment{0};
-const Mn::GL::Framebuffer::ColorAttachment ObjectIdBuffer =
+const Mn::GL::Framebuffer::ColorAttachment ObjectIdTextureColorAttachment =
     Mn::GL::Framebuffer::ColorAttachment{1};
 const Mn::GL::Framebuffer::ColorAttachment UnprojectedDepthBuffer =
```

Comment:
Why rename just one and not all? `ColorAttachment` seems like a clearer name, regardless of whether the underlying object is a renderbuffer or a texture.